go.mod

```diff
@@ -349,3 +349,14 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.13.9 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+require github.com/moby/sys/mountinfo v0.6.2 // indirect
+
+require github.com/google/go-containerregistry v0.12.1
```

Comment:
Why this line ended up here? I'd expect it with other imports. Have you manually added it?

Reply:
No, it was done by `go get`. I can change the file manually if you want, but this is how it ended up just with go tooling.

Reply:
Then it's fine, leave it as it is.